### DISCUSSION TITLE
Option to apply color threshold to entire LCD gauge

### DISCUSSION THREAD

**Discussion by jtommi** [grafana/grafana#37297] [Dashboards, Panels & Visualizations]
This is linked to [my previous post in the forum](https://community.grafana.com/t/solved-bar-gauge-apply-threshold-color-to-entire-bar/33274).

## What
An option to apply color threshold to the entire length of the LCD gauge, not only the part within the threshold

## Background
I prefer the look of the LCD gauge over the basic gauge, but don't like that the thresholds don't color the whole bar (as it does with the basic gauge.

## Example
The use case I come across all the time is some KPI where something should reach 100%.
The goal is 100%, 90% is ok, 70-90% is meh and <70% is bad.
So I define the base threshold as red, 70% as yellow or orange and 90% as green.
![image](https://user-images.githubusercontent.com/33985391/127303574-64dd5867-8f4e-4ae3-bed8-ca4e551d2455.png)

And so even if the 100% goal is reached, most of the gauge is colored in "negative" colors instead of being completely green:
![image](https://user-images.githubusercontent.com/33985391/127303061-8bf68be0-98a4-4516-b997-ad0fbc14e648.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
